plan/plans/index_test.go

```diff
@@ -65,8 +50,9 @@ func (p *testIndexSuit) SetUpSuite(c *C) {
 	store, err := tidb.NewStore(tidb.EngineGoLevelDBMemory)
 	c.Assert(err, IsNil)
 	p.store = store
-	p.vars = map[string]interface{}{}
-	p.txn, _ = p.store.Begin()
+	se, _ := tidb.CreateSession(store)
```

Comment:
I mean in line 54, `c.Assert(err, IsNil)` is needless if line 53 does not have an error return value.